gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/TTimestampWrapper.java

```diff
@@ -132,4 +156,57 @@ public int getMibiecIntValue() {
     public int getAntcfinalIntValue() {
         return timestamp.getANTCFinal().getV().intValue();
     }
+
+    //
+
+    Map<String, Boolean> getExportCornerActiveForCountryMap() {
+        if (exportingCountryMap == null) {
+            exportingCountryMap = new HashMap<>();
+            List<TCalculationDirection> calculationDirections = timestamp.getCalculationDirections().get(0).getCalculationDirection();
+            calculationDirections.forEach(tCalculationDirection -> {
+                if (tCalculationDirection.getInArea().getV().equals(eicCodesConfiguration.getItaly())) {
+                    exportingCountryMap.put(tCalculationDirection.getOutArea().getV(), false);
+                } else if (tCalculationDirection.getOutArea().getV().equals(eicCodesConfiguration.getItaly())) {
+                    exportingCountryMap.put(tCalculationDirection.getInArea().getV(), true);
+                }
+            });
+            exportingCountryMap.put(toEic(Country.IT), true);
```

Comment:
Not sure we can define that. I'm not sure it makes sense because export corner is defined towards Italian exchange.